articles/user-guide/libraries/numerics/toc.yml

```diff
@@ -1,4 +1,2 @@
-- name: Installation and validation
-  href: installation.md
 - name: Using the numerics library
-  href: numerics.md
\ No newline at end of file
+  href: numerics.md
```

Comment:
Verifying that this file was deleted, because the redirect file needs to be updated. Should the redirect just go to the Intro to Numerics topic?

Reply:
I think it was just the installation and validation page that was deleted?

Reply:
Oops, comment went on the wrong line - yes, just the installation and validation page.

Reply:
Sounds good, will set up the redirect, then.